<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionNewSetup"/>
    <addaction name="actionLoadSetup"/>
    <addaction name="separator"/>
    <addaction name="actionSaveSetup"/>
    <addaction name="actionRenameSetup"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionProcessForm"/>
    <addaction name="actionDispersionForm"/>
    <addaction name="actionPolingForm"/>
    <addaction name="separator"/>
    <addaction name="actionPEF"/>
    <addaction name="actionPMF"/>
    <addaction name="actionJSA"/>
   </widget>
   <widget class="QMenu" name="menuExecute">
    <property name="title">
     <string>Execute</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionOptimize"/>
   </widget>
   <addaction name="menuSetup"/>
   <addaction name="menuExecute"/>
   <addaction name="menuWindows"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewSetup">
   <property name="text">
    <string>New Setup</string>
   </property>
  </action>
  <action name="actionLoadSetup">
   <property name="text">
    <string>Load Setup</string>
   </property>
  </action>
  <action name="actionSaveSetup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Setup</string>
   </property>
  </action>
  <action name="actionRenameSetup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rename Setup</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionProcessForm">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Process Form</string>
   </property>
  </action>
  <action name="actionDispersionForm">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dispersion Form</string>
   </property>
  </action>
  <action name="actionPolingForm">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Poling Form</string>
   </property>
  </action>
  <action name="actionPEF">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PEF</string>
   </property>
  </action>
  <action name="actionPMF">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PMF</string>
   </property>
  </action>
  <action name="actionJSA">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>JSA</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionOptimize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Optimize</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
